apps/meteor/tests/e2e/page-objects/fragments/home-sidenav.ts

```diff
@@ -47,6 +47,59 @@ export class HomeSidenav {
 		await this.page.locator(`[data-qa="sidebar-item-title"] >> text="${name}"`).first().click();
 	}
 
+	async openDisplayOptions(): Promise<void> {
+		await this.page.locator('[title="Display"]').click();
+	}
+
+	async selectOrderByName(): Promise<void> {
+		const label = this.page.locator('text=⦇Name >> label');
+
+		const inputIsChecked = await label.locator('input').isChecked();
+		if (!inputIsChecked) {
+			await label.click();
+
+			// Wait for child change
+			await this.page.locator('.rooms-list .rc-scrollbars-view > div > div').evaluate((div) => {
```

Comment:
you can't put a data qa on here. Because if have any changes on the FE this test may break.

Reply:
I wasn't able to add the data-qa because this field comes from the Library